reporters/kamon-prometheus/src/main/scala/kamon/prometheus/ScrapeDataBuilder.scala

```diff
@@ -107,6 +120,33 @@ class ScrapeDataBuilder(prometheusConfig: PrometheusSettings.Generic, environmen
     })
   }
 
+  private def appendDistributionMetricAsSummary(metric: MetricSnapshot.Distributions): Unit = {
+    val unit = metric.settings.unit
+    val normalizedMetricName = normalizeMetricName(metric.name, unit)
+
+    if(metric.description.nonEmpty)
+      append("# HELP ").append(normalizedMetricName).append(" ").append(metric.description).append("\n")
+
+    append("# TYPE ").append(normalizedMetricName).append(" summary").append("\n")
+
+    metric.instruments.foreach(instrument => {
+      if(instrument.value.count > 0) {
+        appendSummaryQuantiles(
+          normalizedMetricName,
+          instrument.tags,
+          instrument.value,
+          unit,
+          prometheusConfig.summarySettings.quantiles
+        )
+        appendTimeSerieValue(normalizedMetricName, instrument.tags, format(instrument.value.max), "_max")
+        appendTimeSerieValue(normalizedMetricName, instrument.tags, format(instrument.value.min), "_min")
```

Comment:
I think both `_min` and `_max` are super valuable pieced of data, but they are not part of the Prometheus spec. Do you have any idea of whether having those fields could affect Prometheus' behaviour?

Reply:
your right
Couldn't find anything about min and max...
removed both just to be save (user can allways get max with the configuration of the 1.0 quantile)